test/core/sup_tree_core/periodic_log/reduction_log_builder_test.exs

```diff
@@ -3,9 +3,23 @@ defmodule AntikytheraCore.PeriodicLog.ReductionBuilderTest do
 
   @dir Path.join([__DIR__, "..", "..", "..", "..", "_build", "test", "log", "antikythera"]) |> Path.expand()
 
-  test "should exists" do
-    assert Process.whereis(ReductionBuilder) != nil
-    send(ReductionBuilder, :timeout)
+  test "should exist" do
+    pid = Process.whereis(ReductionBuilder)
+    assert pid != nil
     assert File.exists?(Path.join(@dir, "reduction.log.gz"))
+
+    state1 = :sys.get_state(pid)
+    build_state1 = state1.build_state
+    # NOTE: The below `accert` statement won't pass. The process probably left its initial state
```

Comment:
```suggestion
    # NOTE: The below `assert` statement won't pass. The process probably left its initial state
```

Reply:
Thank you!
I committed your suggestion.
https://github.com/access-company/antikythera/pull/93/commits/5e7b0d6c46d4c1d11ad549028946f6568908a789